app/src/main/java/com/framgia/moviedb_31/screen/movieDetail/MovieDetailViewModel.java

```diff
@@ -0,0 +1,74 @@
+package com.framgia.moviedb_31.screen.movieDetail;
+
+import android.databinding.BaseObservable;
+import android.databinding.ObservableField;
+import com.framgia.moviedb_31.R;
+import com.framgia.moviedb_31.data.model.Actor;
+import com.framgia.moviedb_31.data.model.Movie;
+import com.framgia.moviedb_31.data.model.Production;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MovieDetailViewModel extends BaseObservable {
+    public ObservableField<Movie> mMovie = new ObservableField<>();
+    public ObservableField<MovieDetailProductionAdapter> mAdapterObservableField =
+            new ObservableField<>();
+    public ObservableField<MovieDetailActorAdapter> mActorAdapterObservableField =
+            new ObservableField<>();
+
+    private MovieDetailProductionAdapter mMovieDetailProductionAdapter;
+    private MovieDetailActorAdapter mMovieDetailActorAdapter;
+
+    MovieDetailViewModel() {
+        mMovieDetailProductionAdapter = new MovieDetailProductionAdapter();
+        mMovieDetailActorAdapter = new MovieDetailActorAdapter();
+        initData();
+        setDataItemProduction();
+        setDataItemActor();
+    }
+
+    private void initData() {
+        Movie movie = new Movie();
+        movie.setNameMovie("Dead Pool");
+        movie.setPoster(String.valueOf(R.drawable.poster2));
+        movie.setContent(
+                "Deadpool xoay quanh anh chàng Wade Wilson, một người bị ung thư vô phương cứu "
+                        + "chữa bị thí nghiệm trở thành dị nhân với khả năng phục hồi siêu tốc "
+                        + "giống Wolverine ...");
+        movie.setRating("8.2");
+        movie.setDate("30/10/2018");
+        movie.setID("3687194");
+        mMovie.set(movie);
+    }
+
+    private void setDataItemProduction() {
+        List<Production> productionList = new ArrayList<>();
+        for (int i =0;i<10;i++){
+            Production production = new Production();
+            production.setID("52434");
+            production.setNameProduction("Marvel");
+            production.setPoster(String.valueOf(R.drawable.poster2));
+            productionList.add(production);
+        }
+        mMovieDetailProductionAdapter.updateAdapter(productionList);
+        mAdapterObservableField.set(mMovieDetailProductionAdapter);
+    }
+
+    private void setDataItemActor() {
+        List<Actor> actorList = new ArrayList<>();
+        for (int i=0;i<10;i++){
```

Comment:
sao lại có for ở trong get data này nưã

Reply:
a ơi e set dữ liệu ảo để làm giao diện thôi a